enterprise/server/backends/gcs_cache/gcs_cache.go

```diff
@@ -76,9 +77,11 @@ func NewGCSCache(bucketName, projectID string, ageInDays int64, opts ...option.C
 		return nil, err
 	}
 	g := &GCSCache{
-		gcsClient: gcsClient,
-		projectID: projectID,
-		ttlInDays: ageInDays,
+		gcsClient:          gcsClient,
+		projectID:          projectID,
+		ttlInDays:          ageInDays,
+		cacheType:          resource.CacheType_CAS,
+		remoteInstanceName: "",
```

Comment:
nit: "" is the default value for strings, there's no need to explicitly specify it when initializing structs

Reply:
These fields are temporary and will be deleted when WithIsolation is deleted so just going to leave for now